docs/content/operator/common/archives.mdx

```diff
@@ -95,24 +105,32 @@ state-archive-read-config:
 </Tabs>
 
 
-Even though these buckets are publicly readable, you need to make sure to properly grant the correct policies to read them via AWS, for example:
-```
+<!-- ### Configure AWS IAM Permissions for Archive Access
+
+To securely access the public archive buckets while following AWS best practices, you need to configure proper IAM permissions. Even though the archive buckets are publicly readable, it's recommended to create a specific IAM policy with the minimum required permissions.
+
+Here's an example IAM policy that grants read-only access to the IOTA mainnet archive bucket:
+
+```json
 {
-    "Version": "2012-10-17",
-    "Statement": [
-        {
-            "Action": [
-                "s3:ListBucket",
-                "s3:GetObject",
-            ],
-            "Effect": "Allow",
-            "Resource": [
-                "arn:aws:s3:::iota-mainnet-archive/*",
-                "arn:aws:s3:::iota-mainnet-archive"
-            ]
-        }
-    ]
+  "Version": "2012-10-17",
+  "Statement": [
+      {
+          "Action": [
+              "s3:ListBucket",  // Allows listing objects in the bucket
+              "s3:GetObject"    // Allows reading objects from the bucket
+          ],
+          "Effect": "Allow",
+          "Resource": [
+              "arn:aws:s3:::iota-mainnet-archive/*",
+              "arn:aws:s3:::iota-mainnet-archive"
+          ]
+      }
+  ]
 }
 ```
 
+For testnet access, replace `mainnet` with `testnet` in the ARN. This policy should be attached to the IAM user or role that your node uses to access the archive. -->
```

Comment:
Maybe we should add a network picker here.

Reply:
Added in c830b16e6c9d4a3b329da01ebf17bfd03d816656